test-requirements.txt

```diff
@@ -10,5 +10,6 @@ pytest-cov
 pytest-xdist
 python-dateutil==2.8.0
 sagemaker>=1.3.0
+smdebug==0.4.13
```

Comment:
Do we want this locked to this version?

Reply:
I checked with Edward - he recommends locking this to a version for now. This is consistent with the requirements.txt file.

Reply:
What Rizwan said. I will test/review smdebug and update.